scispacy/candidate_generation.py

```diff
@@ -11,13 +11,39 @@
 from nmslib.dist import FloatIndex
 
 from scispacy.file_cache import cached_path
-from scispacy.umls_utils import UmlsKnowledgeBase
+from scispacy.umls_utils import KnowledgeBase, UmlsKnowledgeBase, MeshKnowledgeBase
 
-DEFAULT_PATHS = {
-    "ann_index": "https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/data/linking_model/nmslib_index.bin",
-    "tfidf_vectorizer": "https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/data/linking_model/tfidf_vectorizer.joblib",  # noqa
-    "tfidf_umls_vectors": "https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/data/linking_model/tfidf_vectors_sparse.npz",  # noqa
-    "concept_aliases_list": "https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/data/linking_model/concept_aliases.json",  # noqa
+
+class LinkerPaths(NamedTuple):
+    ann_index: str
+    tfidf_vectorizer: str
+    tfidf_vectors: str
+    concept_aliases_list: str
+
+
+UmlsLinkerPaths = LinkerPaths(
+    ann_index="https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/data/linking_model/nmslib_index.bin",
+    tfidf_vectorizer="https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/data/linking_model/tfidf_vectorizer.joblib",  # noqa
+    tfidf_vectors="https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/data/linking_model/tfidf_vectors_sparse.npz",  # noqa
+    concept_aliases_list="https://s3-us-west-2.amazonaws.com/ai2-s2-scispacy/data/linking_model/concept_aliases.json",  # noqa
+)
+
+MeshLinkerPaths = LinkerPaths(
```

Comment:
99% sure you did this intentionally and will upload to s3 later, but commenting anyway